tools/block-generator/runner/run.go

```diff
@@ -188,6 +200,16 @@ func (r *Args) run(reportDirectory string) error {
 	}()
 
 	// create conduit config from template
+	var conduitConfigTmpl string
+	switch r.Template {
+	case "file-exporter":
+		conduitConfigTmpl = conduitFileExporterConfigTmpl
+	case "postgres-exporter":
+		conduitConfigTmpl = conduitPostgresConfigTmpl
+	default:
+		return fmt.Errorf("unknown template type: %s", r.Template)
+	}
```

Comment:
In the future we could change the default case with an attempt to read a file at `r.Template` as a way to provide a custom template.

Reply:
Good idea!

Reply:
added a TODO: 00ddd92dd80302427a735217a52eec6cb12c84bb